src/index.ts

```diff
@@ -20,126 +34,179 @@ async function main() {
+          .option("public_input", {
+            alias: "public_input",
+            describe:
+              "public input of the proof, inputs must have format (0x)[0-f]*:(i64|bytes|bytes-packed) and been separated by spaces (eg: 0x12:i64 44:i64 32:i64).",
+            type: "string",
+            nargs: 1,
+          })
+          .option("private_input", {
+            alias: "private_input",
+            describe: "private currently not supported",
+            type: "string",
+            nargs: 1,
+          });
       },
       // Handler for your command
       async function (argv: any) {
         console.log("Begin adding prove task for ", argv.i, argv.public_input);
-        await addProvingTask(argv.r, argv.u, argv.i,
+        await addProvingTask(
+          argv.r,
+          argv.u,
+          argv.i,
           argv.public_input ? argv.public_input : "",
           argv.priv_input ? argv.priv_input : "",
-          argv.priv);
+          argv.priv
+        );
       }
-    ).command(
-      'adddeploytask',
-      'Add deploy task\n Example: npx adddeploytask -r \"http://127.0.0.1:8080\" -i \"4CB1FBCCEC0C107C41405FC1FB380799\" -u \"0x278847f04E166451182dd30E33e09667bA31e6a8\" -x "xxxxxxx" -c 5',
+    )
+    .command(
+      "adddeploytask",
+      'Add deploy task\n Example: npx adddeploytask -r "http://127.0.0.1:8080" -i "4CB1FBCCEC0C107C41405FC1FB380799" -u "0x278847f04E166451182dd30E33e09667bA31e6a8" -x "xxxxxxx" -c 5',
       // options for your command
       function (yargs: any) {
-        return yargs.option('i', {
-          alias: 'image',
-          describe: 'image md5',
-          demandOption: "The image md5 is required",  // Required
-          type: 'string',
-          nargs: 1
-        }).option('c', {
-          alias: 'chain_id',
-          describe: 'chain id of the network to deploy',
-          demandOption: "Network id is required",
-          type: 'number',
-          nargs: 1
-        })
+        return yargs
+          .option("i", {
+            alias: "image",
+            describe: "image md5",
+            demandOption: "The image md5 is required", // Required
+            type: "string",
+            nargs: 1,
+          })
+          .option("c", {
+            alias: "chain_id",
+            describe: "chain id of the network to deploy",
+            demandOption: "Network id is required",
+            type: "number",
+            nargs: 1,
+          });
       },
       // Handler for your command
       async function (argv: any) {
         console.log("Begin adding deploy task for ", argv.i, argv.c);
         await addDeployTask(argv.r, argv.u, argv.i, argv.c, argv.priv);
       }
     )
+    .command(
+      "addpayment",
+      'Add payment\n Example: npx addpayment -r "http://127.0.0.1:8080" -t "<transactionhash>" \n\n Example new payment: \n npx addpayment -r "http://127.0.0.1:8080" -p "https://goerli.infura.io/v3/xxxxxxx" -u "YOUR_ADDRESS" -x "YOUR_PRIVATE_KEY" -a "0.00001" ',
```

Comment:
I think I originally set -u as "demandOption", so if add payment do not need it, maybe we can move -u from common "demandOption" to each command which need it?